SUI/SUI/Products/ProductsView.swift

```diff
@@ -0,0 +1,225 @@
+        ZStack {
+            Button {
+                productsViewModel.isFiltersShown = true
+            } label: {
+                Image(systemName: Constants.filterButtonImageName)
+                    .resizable()
+                    .frame(width: 25, height: 25)
+                    .foregroundColor(.white)
+            }
+            NavigationLink(isActive: $productsViewModel.isFiltersShown) {
+                FiltersView()
+            } label: {
+                EmptyView()
+            }
+        }
+        .padding(.trailing, 20)
+    }
+
+    private var totalPriceTextView: some View {
+        Text("\(Constants.totalString)\(productsViewModel.totalPrice)")
+            .font(.system(size: 30, weight: .bold, design: .default))
+            .padding(.bottom, -10)
+    }
+
+    private var backProductView: some View {
+        RoundedRectangle(cornerRadius: 20)
+            .fill(.white)
+            .frame(width: UIScreen.main.bounds.width - 40, height: 110)
+            .shadow(color: .gray, radius: 6)
+            .padding(10)
+    }
+
+    private var rowsView: some View {
+        ForEach(0 ..< productsViewModel.products.count, id: \.self) { item in
+            rowView(item: item)
+        }
+        .padding(.top, 10)
+    }
+
+    private var minusTextView: some View {
+        Text(Constants.minusString)
+            .foregroundColor(.black)
+            .font(.system(size: 30, weight: .bold, design: .default))
+    }
+
+    private var plusTextView: some View {
+        Text(Constants.plusString)
+            .foregroundColor(.black)
+            .font(.system(size: 25, weight: .bold, design: .default))
+    }
+
+    private var oldPriceTextView: some View {
+        Text(Constants.oldPriceString)
+            .strikethrough()
+            .foregroundColor(.lightGray)
+            .font(.system(size: 30, weight: .medium, design: .default))
+    }
+
+    // MARK: - Private methods
+
+    private func rowView(item: Int) -> some View {
+        ZStack {
+            backProductView
+            HStack {
+                Spacer(minLength: 40)
+                productImageView(product: productsViewModel.products[item])
+                Spacer(minLength: 10)
+                VStack {
+                    titleTextView(product: productsViewModel.products[item])
+                    HStack {
+                        minusButtonView(item: item)
+                        productCountTextView(product: productsViewModel.products[item])
+                        plusButtonView(item: item)
+                    }
+                }
+                Spacer(minLength: 20)
+                VStack {
+                    oldPriceTextView
+                    newPriceTextView(product: productsViewModel.products[item])
+                }
+                .padding(.trailing, 20)
+                Spacer()
+            }
+        }
+    }
+
+    private func minusButtonView(item: Int) -> some View {
```

Comment:
```suggestion
    private func makeMinusButtonView(item: Int) -> some View {
```
У методов с аргументами которые создают вью приставка make всегда